src/main.cpp

```diff
@@ -1,5 +1,7 @@
 #include "nvim/nvim.h"
 #include "renderer/renderer.h"
+#include <shellscalingapi.h>
+#include <dwmapi.h>
```

Comment:
I think we should move these two #includes to `target_precompile_headers` in `CMakeLists.txt` and remove them from here, as they are system headers.